firebase-sessions/src/test/kotlin/com/google/firebase/sessions/EventGDTLoggerTest.kt

```diff
@@ -26,16 +26,17 @@ import com.google.firebase.sessions.settings.SessionsSettings
 import com.google.firebase.sessions.testing.FakeFirebaseApp
 import com.google.firebase.sessions.testing.FakeFirebaseInstallations
 import com.google.firebase.sessions.testing.FakeProvider
-import com.google.firebase.sessions.testing.FakeTimeProvider
 import com.google.firebase.sessions.testing.FakeTransportFactory
 import com.google.firebase.sessions.testing.TestSessionEventData
+import kotlinx.coroutines.ExperimentalCoroutinesApi
 import kotlinx.coroutines.asCoroutineDispatcher
 import kotlinx.coroutines.test.runTest
 import org.junit.After
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.robolectric.RobolectricTestRunner
 
+@OptIn(ExperimentalCoroutinesApi::class)
```

Comment:
What is this for?

Reply:
The test I edited calls `runTest`, which is marked as `@ExperimentalCoroutinesApi`. Since I touched the file, I also opted in to suppress the warning.